debezium-server-core/pom.xml

```diff
@@ -172,8 +172,9 @@
                 </executions>
             </plugin>
             <plugin>
-                <groupId>org.jboss.jandex</groupId>
+                <groupId>io.smallrye</groupId>
                 <artifactId>jandex-maven-plugin</artifactId>
+                <version>3.1.7</version>
```

Comment:
this is not needed, we need to merge https://github.com/debezium/debezium/pull/6588 first